src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Person as PersonType } from '../types/Person';
+import { Person } from '../components/Person';
+
+interface Props {
+  people: PersonType[];
+  activePerson: PersonType | null;
+  onPersonSelect: (person: PersonType) => void;
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
```

Comment:
The `activePerson` and `onPersonSelect` props are defined in the `Props` interface but are not used in the `PeopleTable` component. Consider removing these props if they are not needed, or implement their usage if they are intended to be part of the component's functionality.